examples/tornjak/values-ingress.yaml

```diff
@@ -0,0 +1,64 @@
+global:
+  spire:
+    jwtIssuer: https://oidc-discovery.$APP_SUBDOMAIN
```

Comment:
I know what your trying to do and why, but it feels really wrong to try and template out values passed to a templating engine used for templating k8s objects.

There a feature we could add to handle some of this better? Been kind of on my todo list for a while. Maybe an globals.app_subdomain or something?